ms/applink.c

```diff
@@ -132,6 +132,15 @@ OPENSSL_Applink(void)
     return OPENSSL_ApplinkTable;
 }
 
+extern __declspec(dllexport)
+void 
+# if defined(__BORLANDC__)
+__stdcall
+# else
+__cdecl
+# endif
+OPENSSL_SetApplink(void** (*custom)());
```

Comment:
Whoops -- you're right. Would it be appropriate to just remove the __BORLANDC__ test? Would that break bordland-c builds?